wvd-templates/Update existing WVD host pool/mainTemplate.json

```diff
@@ -302,78 +357,113 @@
                 "platformFaultDomainCount": 2
             },
             "sku": {
-                "name": "[variables('avSetSKU')]"
+                "name": "Aligned"
```

Comment:
Same comment as the Create mainTemplate - I think this needs to be "Classic" if rdshManagedDisks is true, see avSetSKU in the previous version of this file.

Reply:
Actually this is fine as is since rdshUseManagedDisks is deleted from this file, but I 'll leave this comment here since I assume you are planning on adding unmanaged disk support back it back.